src/chat/chatSelectors.js

```diff
@@ -18,12 +18,20 @@ import {
   isStreamNarrow,
   isHomeNarrow,
   homeNarrow,
-  allPrivateNarrowStr,
 } from '../utils/narrow';
 import { shouldBeMuted } from '../utils/message';
 import { countUnread } from '../utils/unread';
 import { NULL_MESSAGE, NULL_USER, NULL_SUBSCRIPTION } from '../nullObjects';
 
+const specialNarrow = (operand: string): Narrow => [
```

Comment:
Just duplicating code is not a good solution. What happens the next time we got this error? Duplicate more? Slippery slope, my friend.